reporting/impl/src/main/resources/reports/templates/application_list.ftl

```diff
@@ -223,9 +218,9 @@
         </#if>
 
         <div style="width: 100%; text-align: center">
-            <a href="reports/windup_ruleproviders.html">Executed rules overview</a>
+            <a href="reports/windup_ruleproviders.html">Rule providers execution overview</a>
```

Comment:
This doesn't include the title change, not sure why, but if that should be consistent I will change that.